public/js/tas.js

```diff
@@ -1,10 +1,14 @@
 $(function(){
+  // listens for screen size inserts an <hr> tag
+  if ($(window).width() > 600) {
```

Comment:
This bit would be better done in css as this doesn't account for users changing window size.